src/languages/en.ts

```diff
@@ -192,6 +192,7 @@ export default {
             phoneNumber: `Please enter a valid phone number, with the country code (e.g. ${CONST.EXAMPLE_PHONE_NUMBER})`,
             fieldRequired: 'This field is required.',
             characterLimit: ({limit}: CharacterLimitParams) => `Exceeds the maximum length of ${limit} characters`,
+            characterLimitExceedCounter: ({length, limit}) => `Characters limit exceeded (${length}/${limit})`,
```

Comment:
```suggestion
            characterLimitExceedCounter: ({length, limit}) => `Character limit exceeded (${length}/${limit})`,
```

I think this is more gramatically correct...?

Reply:
Yes, correct.